### PR TITLE
Test request token logic a bit more

### PR DESCRIPTION
**Pull request by ChristophWurst**
Just trying to get code into more of a functional style so we can test it more easily without nasty setup work on global variables.

cc @nextcloud/javascript

### CODE REVIEW

**Review by juliusknorr — APPROVED** [review]

File: core/src/OCP/initialstate.js [hunk 1]
```diff
@@ -1,29 +0,0 @@
-/*
```
Comment (rullzer) [inline core/src/OCP/initialstate.js]: Does this need to die?

Comment (ChristophWurst) [reply core/src/OCP/initialstate.js]: it's not used. This is a leftover from https://github.com/nextcloud/server/pull/17390/files#diff-6e6396c962319cd2299df9376a89c7d0

Comment (rullzer) [reply core/src/OCP/initialstate.js]: All good then. carry on!

**Review by rullzer — APPROVED** [review]

**Comment by rullzer** [comment]
Does need to have the assets build ;)

**Pull request merged**